test/integration/segmentation_camera_plugin.cc

```diff
@@ -265,21 +270,26 @@ void SegmentationCameraSensorTest::ImagesWithBuiltinSDF(
   uint32_t rightIndex = (rightProj.Y() * width + rightProj.X()) * channels;
   uint32_t middleIndex = (middleProj.Y() * width + middleProj.X()) * channels;
 
-  // test
-  g_counter = 0;
-
   uint8_t leftLabel =   g_buffer[leftIndex];
   uint8_t rightLabel =  g_buffer[rightIndex];
   uint8_t middleLabel = g_buffer[middleIndex];
 
   // check the label
-  EXPECT_EQ(leftLabel, 1);
-  EXPECT_EQ(middleLabel, 2);
-  EXPECT_EQ(rightLabel, 1);
+  EXPECT_EQ(leftLabel, leftBoxLabel);
+  EXPECT_EQ(middleLabel, middleBoxLabel);
+  EXPECT_EQ(rightLabel, rightBoxLabel);
 
+  // check a pixel between 2 boxes & a pixel below a box
+  uint32_t betweenBoxesIndex = (120 * width + 230) * channels;
+  uint32_t belowBoxesIndex = (200 * width + 280) * channels;
   // check if the first pixel(background) = the background label
   uint8_t background = g_buffer[0];
+  uint8_t betweenBoxes = g_buffer[betweenBoxesIndex];
+  uint8_t belowBoxes = g_buffer[belowBoxesIndex];
```

Comment:
```suggestion
  // check a pixel between 2 boxes & a pixel below a box
  uint32_t betweenBoxesIndex = (120 * width + 230) * channels;
  uint32_t belowBoxesIndex = (200 * width + 280) * channels;
  uint8_t betweenBoxes = g_buffer[betweenBoxesIndex];
  uint8_t belowBoxes = g_buffer[belowBoxesIndex];
  // check if the first pixel(background) = the background label
  uint8_t background = g_buffer[0];
```
nit: re-arrange code to avoid confusion with the comments